CHANGELOG.md

```diff
@@ -2514,6 +2514,17 @@ Other minor changes
   flip′ : (A → B → C) → (B → A → C)
   ```
 
+* Added new proofs to `Data.List.Properties`
+  ```agda
+  cartesianProductWith-zeroˡ       : ∀ (f : A → B → C) ys → cartesianProductWith f [] ys ≡ []
```

Comment:
Thanks for the entries!

We usually only provide the "interesting" part of the type in the CHANGELOG to avoid overwhelming the reader with too much detail, (e.g. `cartesianProductWith f [] ys ≡ []` and `foldr f x (map g xs) ≡ foldr (g -⟨ f ∣) x xs`). Any chance you could tweak them slightly?

Reply:
Sorry, my bad!

Reply:
No worries, no reason why you should have known :smile: